src/import/sst_service.rs

```diff
@@ -94,6 +96,106 @@ where
         let p = sst_meta_to_path(meta)?;
         Ok(slots.remove(&p))
     }
+
+    fn check_write_stall(&self) -> Option<errorpb::Error> {
+        if self.importer.get_mode() == SwitchMode::Normal
+            && self
+                .engine
+                .ingest_maybe_slowdown_writes(CF_WRITE)
```

Comment:
Why only check for write cf?

Reply:
Because the number of files is larger usualy